src/main/webapp/data/userMapData.json

```diff
@@ -1883,6 +1883,7 @@
       "UNC Pembroke",
       "Union Mine High School",
       "United States Military Academy Preparatory School",
+      "Univeristy of North Georgia",
```

Comment:
Let's use this opportunity to fix this typo.

Reply:
Fixed two typos 👍